sdk/src/wallet/account/operations/transaction/sign_transaction.rs

```diff
@@ -60,7 +60,8 @@ where
                         self.emit(
                             self.details().await.index,
                             WalletEvent::TransactionProgress(TransactionProgressEvent::PreparedTransactionHash(
-                                prefix_hex::encode(prepared_transaction_data.transaction.hash()),
+                                // TODO: Should this be the signing_hash?
```

Comment:
Yes. Please rename `TransactionProgressEvent::PreparedTransactionHash` to `TransactionProgressEvent::PreparedTransactionSigningHash`, also in the bindings.